SweetTweetApp/src/main/webapp/resourses/js/View/View.js

```diff
@@ -0,0 +1,54 @@
+class View {
+    addPostsToNewsLine(posts, currentAuthor){
+        let postsWrapper = document.getElementById("news-line");
```

Comment:
Just a piece of advice: try to use [document.querySelector](https://developer.mozilla.org/ru/docs/Web/API/Document/querySelector) since it selects item base on css selectors